main/res/xml/preferences.xml

```diff
@@ -352,50 +352,28 @@
                 android:title="@string/init_geokrety" >
                 <PreferenceCategory android:title="@string/settings_settings" >
                     <CheckBoxPreference
-                        android:defaultValue="true"
```

Comment:
You should separate different issues in different PRs, even if one of them contains only one single line. This line is about having GK not active by default and is completely unrelated to the remaining changes about authorization.